tutorials/SDF/car_demo.md

```diff
@@ -0,0 +1,520 @@
+            <ambient>1.0 0.0 0.0 1</ambient>
+            <diffuse>1.0 0.0 0.0 1</diffuse>
+            <specular>1.0 0.0 0.0 1</specular>
+        </material>
+    </visual>
+    <collision name='collision'>
+        <geometry>
+            <cylinder>
+                <radius>0.4</radius>
+                <length>0.2</length>
+            </cylinder>
+        </geometry>
+    </collision>
+</link>
+```
+
+The right wheel is similar to the left wheel except for its position.
+
+### Defining an arbitrary frame
+
+One of the new features of SDF 1.7 is that we can define arbitrary frames. It takes two attributes:
+
+* `name`: the name of the frame
+* `attached_to`: the name of the frame or the link to which this frame is attached.
+
+Let's add a frame for our caster wheel as follows:
+
+```xml
+<frame name="caster_frame" attached_to='chassis'>
+    <pose>0.8 0 -0.2 0 0 0</pose>
+</frame>
+```
+
+We gave our frame name `caster_frame` and attached it to the `chassis` link, then the `<pose>` tag to define the position and orientation of the frame.
+We didn't use the `relative_to` attribute so the pose is with respect to the frame named in the `attached_to` attribute, `chassis` in our case.
+
+### Caster wheel
+
+```xml
+<!--caster wheel-->
+<link name='caster'>
+    <pose relative_to='caster_frame'/>
+    <inertial>
+        <mass>1</mass>
+        <inertia>
+            <ixx>0.016</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>0.016</iyy>
+            <iyz>0</iyz>
+            <izz>0.016</izz>
+        </inertia>
+    </inertial>
+    <visual name='visual'>
+        <geometry>
+            <sphere>
+                <radius>0.2</radius>
+            </sphere>
+        </geometry>
+        <material>
+            <ambient>0.0 1 0.0 1</ambient>
+            <diffuse>0.0 1 0.0 1 1</diffuse>
+            <specular>0.0 1 0.0 1</specular>
+        </material>
+    </visual>
+    <collision name='collision'>
+        <geometry>
+            <sphere>
+                <radius>0.2</radius>
+            </sphere>
+        </geometry>
+    </collision>
+</link>
+```
+
+Our last link is the `caster` and its pose is with respect to the frame `caster_frame` we defined above.
+As you could notice we closed the `pose` tag without defining the position or the orientation; in this case the pose of the link is the same as (identity) the frame in `relative_to`.
+
+In the `<visual>` and `<collision>` tags we defined a different shape `<sphere>` which requires the `<radius>` of the sphere.
+
+### Connecting links together (joints)
+
+We need to connect these links together; here comes the job of `<joint>` tag.
+The joint tag connects two links together and defines how they will move with respect to each other.
+Inside the `<joint>` tag we need to define the two links to connect and their relations(way of movement).
+
+#### Left wheel joint
+
+```xml
+<joint name='left_wheel_joint' type='revolute'> <!--continuos joint is not supported yet-->
```

Comment:
```suggestion
<joint name='left_wheel_joint' type='revolute'>
```

I think we don't need that note about continuous joints. Most likely, people reading this tutorial won't have expectations about what a continuous joint is.

Reply:
@mohamedsayed18 , I still see the comment on the tutorial. Did you intend to remove it?